.github/workflows/latest-versions.yml

```diff
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
 
   # Trigger every Sunday at 08:00
```

Comment:
Could we also change the comment? :)

Reply:
oops, yes!

Reply:
I am not sure why it said 08:00

Github suggestion while editing it said 00:00 UTC, so that's how I changed it.

Reply:
08:00 was indeed wrong, looks great now 👍